cmdhelpers/retrieverconf/retriever_conf.go

```diff
@@ -20,46 +20,45 @@ var DefaultRetrieverConfig = struct {
-	Path           string        `mapstructure:"path"           koanf:"path"`
+	Kind           RetrieverKind `mapstructure:"kind"             koanf:"kind" json:"Kind"`
+	RepositorySlug string        `mapstructure:"repositorySlug"   koanf:"repositoryslug" json:"RepositorySlug"`
+	Branch         string        `mapstructure:"branch"           koanf:"branch" json:"Branch"`
+	Path           string        `mapstructure:"path"             koanf:"path" json:"Path"`
 	// Deprecated: Please use AuthToken instead
-	GithubToken string              `mapstructure:"githubToken"    koanf:"githubtoken"`
-	URL         string              `mapstructure:"url"            koanf:"url"`
-	Timeout     int64               `mapstructure:"timeout"        koanf:"timeout"`
-	HTTPMethod  string              `mapstructure:"method"         koanf:"method"`
-	HTTPBody    string              `mapstructure:"body"           koanf:"body"`
-	HTTPHeaders map[string][]string `mapstructure:"headers"        koanf:"headers"`
-	Bucket      string              `mapstructure:"bucket"         koanf:"bucket"`
-	Object      string              `mapstructure:"object"         koanf:"object"`
-	Item        string              `mapstructure:"item"           koanf:"item"`
-	Namespace   string              `mapstructure:"namespace"      koanf:"namespace"`
-	ConfigMap   string              `mapstructure:"configmap"      koanf:"configmap"`
-	Key         string              `mapstructure:"key"            koanf:"key"`
-	BaseURL     string              `mapstructure:"baseUrl"        koanf:"baseurl"`
-	AuthToken   string              `mapstructure:"token"          koanf:"token"`
+	GithubToken string              `mapstructure:"githubToken"   koanf:"githubtoken" json:"GithubToken"`
+	URL         string              `mapstructure:"url"           koanf:"url" json:"URL"`
+	Timeout     int64               `mapstructure:"timeout"       koanf:"timeout" json:"Timeout"`
+	HTTPMethod  string              `mapstructure:"method"        koanf:"method" json:"HTTPMethod"`
+	HTTPBody    string              `mapstructure:"body"          koanf:"body" json:"HTTPBody"`
+	HTTPHeaders map[string][]string `mapstructure:"headers"       koanf:"headers" json:"HTTPHeaders"`
+	Bucket      string              `mapstructure:"bucket"        koanf:"bucket" json:"Bucket"`
+	Object      string              `mapstructure:"object"        koanf:"object" json:"Object"`
+	Item        string              `mapstructure:"item"          koanf:"item" json:"Item"`
+	Namespace   string              `mapstructure:"namespace"     koanf:"namespace" json:"Namespace"`
+	ConfigMap   string              `mapstructure:"configmap"     koanf:"configmap" json:"ConfigMap"`
+	Key         string              `mapstructure:"key"           koanf:"key" json:"Key"`
+	BaseURL     string              `mapstructure:"baseUrl"       koanf:"baseurl" json:"BaseURL"`
+	AuthToken   string              `mapstructure:"token"         koanf:"token" json:"AuthToken"`
```

Comment:
Do you really need to add the `json` tag in the struct?
What is the reason do you need it?